### PR TITLE
shortcircuit DB lookup

### PR DESCRIPTION
**Pull request by snopoke**
@dimagi/scale-team 

This will only help if all the app reports are static. There is no caching here so this get's called at least once per restore request.

### CODE REVIEW

**Review by nickpell — APPROVED** [review]

**Review by emord — APPROVED** [review]

File: settings.py [hunk 1]
```diff
@@ -1277,6 +1277,8 @@
 
 SYNCLOGS_DB = 'synclogs'
 
+META_DB = 'meta'
+
```
Comment (dannyroberts) [inline settings.py]: Is this related to the other commit or did it just kind of sneak its way in here?

**Review by dannyroberts — COMMENTED** [review]
I love that a `get_db()` call (which I think makes just a `HEAD` request to couch?) might matter now.

**Comment by dannyroberts** [comment]
(Also we might want to patch `get_db()` somehow to not make a request to couch if it currently does. That request is (if my memory serves) is just to see if the database exists, but of course all of these databases exist!)

File: settings.py (hunk 1 above)
Comment (snopoke) [inline settings.py]: yea, snuck it in

**Comment by snopoke** [comment]
@dannyroberts this isn't trying to get rid of calls to `get_db` but rather to `get_doc`. 

Also I don't think `Document.get_db` will do a HEAD request except for the first time for that DB since couchdbkit keeps a global list of databases. If we use `dimagi.utils.couch.database.get_db` or `couch_config.get_db` then it will do a HEAD request

**Comment by dannyroberts** [comment]
`get_docs`, right? I thought you just changed `get_docs` in another PR to short circuit if it was passed an empty list?

**Comment by snopoke** [comment]
I did, I guess this is a double short circuit then

**Pull request merged**